mlflow/h2o.py

```diff
@@ -25,6 +25,15 @@ def save_model(h2o_model, path, conda_env=None, mlflow_model=Model(), settings=N
     :param h2o_model: H2O model to be saved.
     :param path: Local path where the model is to be saved.
     :param mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
+
+    >>> import mlflow
```

Comment:
I don't think we need the import mlflow statement.

Reply:
Yeah, will remove it; redundant.